### PR TITLE
ci: Add circleci.yml

### PR DESCRIPTION
**Pull request by shenli**
@siddontang @huachaohuang  PTAL
Please add pingcap/pd on https://circleci.com

### CODE REVIEW

**Comment by siddontang** [comment]
LGTM

**Comment by shenli** [comment]
@overvenus PTAL

File: README.md [hunk 1]
```diff
@@ -3,6 +3,7 @@
 [![Build Status](https://travis-ci.org/pingcap/pd.svg?branch=master)](https://travis-ci.org/pingcap/pd)
 [![Go Report Card](https://goreportcard.com/badge/github.com/pingcap/pd)](https://goreportcard.com/report/github.com/pingcap/pd)
 [![Coverage Status](https://coveralls.io/repos/github/pingcap/pd/badge.svg?branch=master)](https://coveralls.io/github/pingcap/pd?branch=master)
+[![CircleCI Build Status](https://circleci.com/gh/pingcap/pd.svg?style=shield&circle-token=:circle-token)](https://circleci.com/gh/pingcap/pd)
```
Comment (overvenus) [inline README.md]: It seems that we don't need a `circle-token`.
```
[![CircleCI Build Status](https://circleci.com/gh/pingcap/pd.svg?style=shield)](https://circleci.com/gh/pingcap/pd)
```

**Comment by shenli** [comment]
@overvenus PTAL

**Comment by overvenus** [comment]
LGTM

**Pull request merged**